eiffel-vocabulary/EiffelTestCaseTriggeredEvent.md

```diff
@@ -251,6 +251,7 @@ __Description:__ A URI pointing at a location from where the schema used when cr
 
 | Version | Introduced in | Changes |
 | ------- | ------------- | ------- |
+| 3.3.0 | Current version | Add SCS and SCC as legal target for IUT in TCT (see [Issue 317](https://github.com/eiffel-community/eiffel/issues/317)). |
```

Comment:
Have any good idea what to write here?

Reply:
The big table on https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/event-schemas.md specifies that "No edition set" should be used. That's nonsense, hence #326.